src/NuGet.Services.EndToEnd/Support/TestSettings.cs

```diff
@@ -106,6 +106,28 @@ public static async Task<TestSettings> CreateLocalTestConfigurationAsync(string
             throw new ArgumentException($"{configurationName} is not one of the local test supported configurations!");
         }
 
+        /// <summary>
+        /// Override base URLs for regular (Semver 1) registration to be used in Azure Search tests.
+        /// </summary>
+        /// <remarks>
+        /// Azure search service has registration root supplied through its own configuration
+        /// that it uses it responses (instead of taking one from the service index file). This makes
+        /// switching to (and testing) the alternative registration hives as service index information
+        /// and search service configuration will not align at all times. This setting allows providing
+        /// overrides for expected registration roots for Search Service tests.
+        /// </remarks>
+        public string[] SearchBaseRegistrationRoots { get; set; }
```

Comment:
Why not `BaseUrls` instead of `Roots`? This aligns with other properties in the config better.

Also `List<string>` would align more with the mutability properties on this config class.